assets/js/components/HostDetails/StatusPill.jsx

```diff
@@ -1,27 +1,46 @@
 import React from 'react';
 import classNames from 'classnames';
 
+import { EOS_LENS_FILLED } from 'eos-icons-react';
+
 import Pill from '@components/Pill';
 
 function StatusPill({ className, children, heartbeat }) {
   switch (heartbeat) {
     case 'passing':
       return (
         <Pill
-          className={classNames(className, 'bg-jungle-green-500 text-gray-100')}
+          className={classNames(
+            className,
+            'bg-gray-200 text-gray-500 items-center'
+          )}
         >
-          {children}: running
+          {children}:
+          <EOS_LENS_FILLED size="base" className="fill-jungle-green-500 mx-1" />
+          running
         </Pill>
       );
     case 'critical':
       return (
-        <Pill className={classNames('bg-red-200 text-red-800', className)}>
-          {children}: not running
+        <Pill
+          className={classNames(
+            'bg-gray-200 text-gray-500 items-center',
+            className
+          )}
+        >
+          {children}:
+          <EOS_LENS_FILLED size="base" className="fill-red-500 px-2" />
+          not running
         </Pill>
       );
     default:
       return (
-        <Pill className={classNames('bg-gray-200 text-gray-800', className)}>
+        <Pill
+          className={classNames(
+            'bg-gray-200 text-gray-500 items-center',
+            className
```

Comment:
Shouldn't the unknown have a grey icon? For consistency

Reply:
I followed @jagabomb suggestions here, I think the reasoning behind this is that a gray icon on a gray background would look bad.